desktop/src/@batch-flask/core/server-error/server-error.ts

```diff
@@ -124,7 +127,13 @@ export class ServerError {
     }
 
     public static fromARM(response: HttpErrorResponse): ServerError {
-        const error = response.error || {};
+        // KLUDGE: Angular's request nests the error inside an error object
+        let error;
+        if (response.error?.error) {
+            error = response.error.error;
+        } else {
+            error = response.error;
```

Comment:
Should this be `response.error || {}` to preserve the previous behavior? Also `response.error.error || {}` above. If `error` ends up as null or undefined, line 139 below will error.